docs/locale/fr/LC_MESSAGES/userguide/parsec_workspaces.po

```diff
@@ -0,0 +1,262 @@
+"`workspace roles <doc_userguide_parsec_workspaces_roles>`."
+msgstr ""
+"Trouvez l'utilisateur avec lequel vous voulez partager l'espace de travail "
+"et sélectionnez son :ref:`rôles d'espace de travail "
+"<doc_userguide_parsec_workspaces_roles>`."
+
+#: ../../userguide/parsec_workspaces.rst:-1
+msgid "Sharing a workspace"
+msgstr "Partager un espace de travail"
+
+#: ../../userguide/parsec_workspaces.rst:79
+msgid ""
+"Depending on the user profile within the organization, some roles may not be "
+"available for the selected user."
+msgstr ""
+"Selon le profil de l'utilisateur au sein de l'organisation, certains rôles "
+"peuvent ne pas être disponibles pour l'utilisateur sélectionné."
+
+#: ../../userguide/parsec_workspaces.rst:82
+msgid ""
+"You can remove access to this workspace by selecting ``Not shared`` for a "
+"given user."
+msgstr ""
+"Vous pouvez supprimer l'accès à cet espace de travail en sélectionnant ``Non "
+"partagé`` pour un utilisateur donné."
+
+#: ../../userguide/parsec_workspaces.rst:89
+msgid "Workspace roles"
+msgstr "Rôles des espaces de travail"
+
+#: ../../userguide/parsec_workspaces.rst:91
+msgid ""
+"The **workspace role** defines what the user is allowed to do within the "
+"workspace. Since the role is specific to the workspace, a user can have "
+"different roles in different workspaces."
+msgstr ""
+"Le **rôle de l'espace de travail** définit ce que l'utilisateur est autorisé "
+"à faire dans l'espace de travail. Comme le rôle est spécifique à l'espace de "
+"travail, un utilisateur peut avoir différents rôles dans différents espaces "
+"de travail."
+
+#: ../../userguide/parsec_workspaces.rst:95
+msgid ""
+"The available roles and what they allow to do are shown in the following "
+"table."
+msgstr ""
+"Les rôles disponibles et ce qu'ils permettent de faire sont indiqués dans le "
+"tableau suivant."
+
+#: ../../userguide/parsec_workspaces.rst:101
+msgid "User rights"
+msgstr "Droits de l'utilisateur"
+
+#: ../../userguide/parsec_workspaces.rst:102
+msgid "Reader"
+msgstr "Lecteur"
+
+#: ../../userguide/parsec_workspaces.rst:103
+msgid "Contributor"
+msgstr "Contributeur"
+
+#: ../../userguide/parsec_workspaces.rst:104
+msgid "Manager"
+msgstr "Gérant"
+
+#: ../../userguide/parsec_workspaces.rst:105
+msgid "Owner"
+msgstr "Propriétaire"
+
+#: ../../userguide/parsec_workspaces.rst:106
+msgid "Can view and open files"
+msgstr "Peut visualiser et ouvrir des fichiers"
+
+#: ../../userguide/parsec_workspaces.rst:107
+#: ../../userguide/parsec_workspaces.rst:108
+#: ../../userguide/parsec_workspaces.rst:109
+#: ../../userguide/parsec_workspaces.rst:110
+#: ../../userguide/parsec_workspaces.rst:113
+#: ../../userguide/parsec_workspaces.rst:114
+#: ../../userguide/parsec_workspaces.rst:115
+#: ../../userguide/parsec_workspaces.rst:119
+#: ../../userguide/parsec_workspaces.rst:120
+#: ../../userguide/parsec_workspaces.rst:125
+#: ../../userguide/parsec_workspaces.rst:130
+msgid "✅"
```

Comment:
:sweat_smile: We should see if it's possible to exclude emoji from translation to limit the polution